### PR TITLE
Use the Spanish spell checker and improve the English on the way

### PR DESCRIPTION
**Pull request by palvarez89**

### CODE REVIEW

File: locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -814,7 +814,7 @@ msgid "Let's play some drinking games!"
 msgstr "¡Juegos para beber!"
 
 #: web/templates/index.html:25
-msgid "Roll a die"
+msgid "Roll a dice"
```
Comment (dnaranjo89) [inline locale/es/LC_MESSAGES/django.po]: The singular of *dice* is *die*, isn't it?

File: locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1039,7 +1039,7 @@ msgstr "O"
 
 #: web/templates/profile.html:53
 msgid "Show only your public draws"
-msgstr "Mostar solo tiradas públicas"
+msgstr "Mostra solo tiradas públicas"
```
Comment (dnaranjo89) [inline locale/es/LC_MESSAGES/django.po]: *Mostrar* o *Muesta*, but not *Mostra*

File: web/templates/index.html [hunk 3]
```diff
@@ -22,7 +22,7 @@
         <div class="col-sm-6">
             {% include "snippets/snippet_draw_box.html" with draw_type="coin" draw_icon="img/draw_icons/coin.png" title=_("Flip a coin") help=_("Afraid of taking the wrong decision? Don't hesitate and throw the coin!") %}
             {% include "snippets/snippet_draw_box.html" with draw_type="card" draw_icon="img/draw_icons/cards.png" title=_("Take a card") help=_("Let's play some drinking games!") %}
-            {% include "snippets/snippet_draw_box.html" with draw_type="dice" draw_icon="img/draw_icons/dice.png" title=_("Roll a die") help=_("Don't you have a die? Just roll this one") %}
+            {% include "snippets/snippet_draw_box.html" with draw_type="dice" draw_icon="img/draw_icons/dice.png" title=_("Roll a dice") help=_("Don't you have a die? Just roll this one") %}
```
Comment (dnaranjo89) [inline web/templates/index.html]: So the same with the plurar. Should be *Roll a die* or *Roll dice*

**Comment by dnaranjo89** [comment]
Habeis aceptado ese PR muy rapido.
He dejado comentarios  <https://github.com/etcaterva/EchaloASuerte/pull/418>
https://github.com/etcaterva/EchaloASuerte/pull/418

2015-08-05 0:42 GMT+02:00 Mario Corchero <notifications@github.com>:

File: locale/es/LC_MESSAGES/django.po (hunk 1 above)
Comment (palvarez89) [inline locale/es/LC_MESSAGES/django.po]: Hm.. I thought, and I think I'm right, dice is both, singular and plural. But also I didn't know that "die" exists, we can keep it as it is.

File: locale/es/LC_MESSAGES/django.po (hunk 2 above)
Comment (palvarez89) [inline locale/es/LC_MESSAGES/django.po]: Crap, trying to fix a typo I introduced another :/

**Comment by palvarez89** [comment]
@dnaranjo89 I agree. Also the tests failed :(

We can either revert the second commit or send a patch for the couple of comments. I would prefer reverting the patch given that I don't know what caused Travis to fail.

File: locale/es/LC_MESSAGES/django.po (hunk 1 above)
Comment (dnaranjo89) [inline locale/es/LC_MESSAGES/django.po]: http://grammarist.com/usage/dice-die/

**Comment by Mariocj89** [comment]
test failed because selenium tests are not up to date.

File: locale/es/LC_MESSAGES/django.po (hunk 1 above)
Comment (Mariocj89) [inline locale/es/LC_MESSAGES/django.po]: no body uses die for singular....

**Pull request merged**